ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -133,6 +147,59 @@
   command: systemctl restart iptables
   when: "'rhel7-s390x' in inventory_hostname"
 
+- name: Check if SU is installed on IBMi
+  when: os|startswith("ibmi")
+  stat:
+    path: /QOpenSys/pkgs/bin/su
+  register: su_exists
+
+- name: git clone su on ibmi
+  git:
+    repo: https://github.com/ThePrez/IBMiOSS-utils.git
+    dest: /tmp/IBMiOSS-utils
+  when: su_exists.stat.exists == False and os|startswith("ibmi")
+
+- name: copy su into /QOpenSys/pkgs/bin IBMi
+  copy:
+    src: /tmp/IBMiOSS-utils/su
+    dest: /QOpenSys/pkgs/bin/su
+    remote_src: true
+  when: su_exists.stat.exists == False and os|startswith("ibmi")
+
+- name: copy su-setup into /QOpenSys/pkgs/bin IBMi
+  copy:
+    src: /tmp/IBMiOSS-utils/su-setup
+    dest: /QOpenSys/pkgs/bin/su-setup
+    remote_src: true
+  when: su_exists.stat.exists == False and os|startswith("ibmi")
+
+- name: Check if SU is installed on IBMi
+  when: os|startswith("ibmi")
+  stat:
+    path: /QOpenSys/pkgs/bin/su
+  register: su_exists
+
+# FIXME rewrite jenkins start to use ssh or SBMJOB, so su is not needed.
```

Comment:
I think you have done this, so is su not needed anymore?

Reply:
Correct. We can remove `su`